eo-maven-plugin/src/test/java/org/eolang/maven/DiscoverMojoTest.java

```diff
@@ -149,12 +142,13 @@ void discoversWithVersions(@TempDir final Path tmp) throws IOException {
     }
 
     @Test
-    @Disabled
     void discoversWithSeveralObjectsWithDifferentVersions(
         @TempDir final Path tmp
     ) throws IOException {
+        final CommitHashesMap.Fake hashes = new CommitHashesMap.Fake();
```

Comment:
@volodya-lombrozo What is the point of specifying the exact type here - `CommitHashesMap.Fake`? Maybe it's better to use just `CommitHashesMap`?

Reply:
@maxonfjvipon We can't use `CommitHashesMap` here, but we can use `Map<String, CommitHash>`. Thanks.